app/controllers/spree/orders_controller.rb

```diff
@@ -78,11 +78,7 @@ def update
         discard_empty_line_items
         with_open_adjustments { update_totals_and_taxes }
 
-        if @order == current_order
-          fire_event('spree.order.contents_changed')
-        else
-          @order.update_distribution_charge!
-        end
```

Comment:
we need to remove the consumer from app/models/spree/order.rb you
line 9:
ActiveSupport::Notifications...

Reply:
@luisramos0 Good catch! Removed.